### DISCUSSION TITLE
Disable Auto Module update for user who are satisfied by the current version of modules

### DISCUSSION THREAD

**Discussion by omidRMR** [masterking32/MasterCryptoFarmBot#37] [Ideas]
From time to time, modules stop working after auto upgrading from GitHub.

![image](https://github.com/user-attachments/assets/d7626040-b329-4c02-9064-0b61cd82eaae)

Restarting the program would not help. The only solution (at least for me) is to remove and install the module again. Although continuous upgrades generally improve the module's performance, sometimes it may lead to conflicts. It would be helpful for users to have the option to temporarily disable module auto-updates if they are satisfied with its current state.

**Comment by masterking32** [comment]
create an issue for the module if your error is related to a module not included in the MCF project. Avoid making any code changes in the modules and main project, as this can cause the auto-upgrade to stop working.
I will close any discussions and issues that are not related to the MCF project.
